app/controllers/kudos_controller.rb

```diff
@@ -18,6 +18,7 @@ def new
   end
 
   def edit
+    authorize kudo
     if kudo.employee == current_employee
```

Comment:
[suggestion]
isn't this an authorization? :) Consider moving it into the pundit policies

Reply:
this should be in the policy :)

Reply:
Fixed.